pkg/monitoring/api/prometheus/prometheus.go

```diff
@@ -0,0 +1,64 @@
+package prometheus
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	namespaceAndVMILabels    = []string{"namespace", "vmi"}
+	activePortForwardTunnels = prometheus.NewGaugeVec(
```

Comment:
For discovery reasons it would make sense to move the metrics to `pkg/monitoring/api` I think. Not that it helped me remembering what I exposed there already :D

Reply:
Moved, looks better I agree.